ASP/HTML-ASP.sublime-syntax

```diff
@@ -14,9 +14,10 @@ contexts:
         - match: "%>"
           scope: punctuation.section.embedded.end.asp
           pop: true
-        - match: (').*?(?=%>)
-          scope: comment.line.apostrophe.asp
-          captures:
-            1: punctuation.definition.comment.asp
-        - include: scope:source.asp
+        - match: ''
+          push:
+            - include: scope:source.asp # don't directly push, so that the scope doesn't also contain 'source.asp ' as well as our 'source.asp.embedded.html '
+          with_prototype:
+            - match: "(?=%>)"
+              pop: true
```

Comment:
I just had a quick check with ASP to ensure that it is not valid to split up statements in separate blocks like this:

    <% Dim %><% var_name %>

and fortunately it isn't, so resetting to a new root code context each time is fine when not inside a nested block.